poetry.lock

```diff
@@ -633,8 +633,8 @@ packaging = "*"
 "ruamel.yaml" = "*"
 
 [package.extras]
-test = ["flake8 (>=3.0)", "coverage"]
-dev = ["pypandoc (>=1.4)", "flake8 (>=3.0)", "coverage"]
+dev = ["coverage", "flake8 (>=3.0)", "pypandoc (>=1.4)"]
+test = ["coverage", "flake8 (>=3.0)"]
```

Comment:
Funny thing that happened here, but I guess it looks better in alphabetical order :)